tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -351,7 +351,7 @@ private async Task ExecuteAsync(InvocationContext context)
                             globalCoverage is not null)
                         {
                             // We only report the code coverage percentage if the customer manually sets the 'DD_CIVISIBILITY_CODE_COVERAGE_ENABLED' environment variable according to the new spec.
-                            if (EnvironmentHelpers.GetEnvironmentVariable(Configuration.ConfigurationKeys.CIVisibility.CodeCoverage).ToBoolean() == true)
+                            if ((EnvironmentHelpers.GetEnvironmentVariable(Configuration.ConfigurationKeys.CIVisibility.CodeCoverage) ?? string.Empty).ToBoolean() == true)
```

Comment:
I probably would have done this, but either works 😄 
```suggestion
                            if ((EnvironmentHelpers.GetEnvironmentVariable(Configuration.ConfigurationKeys.CIVisibility.CodeCoverage))?.ToBoolean() == true)
```

Reply:
I like it, to avoid the call 👍🏻

Reply:
done in [9f03604](https://github.com/DataDog/dd-trace-dotnet/pull/5063/commits/9f036044c72606508e8275fb68d45887e71e12d3)